src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -100,6 +106,145 @@ sealed trait Task {
 
 object Task {
 
+  import mesosphere.marathon.api.v2.json.Formats.{ TimestampFormat, PathIdFormat }
```

Comment:
hard to review when huge blocks like this are re-ordered.  maybe you could do the re-ordering in a separate commit?